php/blocks/class-loader.php

```diff
@@ -398,27 +394,20 @@ public function block_template( $name, $type = 'block' ) {
 
 		// So lets fix it.
 		if ( empty( $wp_query ) ) {
-			$wp_query = new \WP_Query(); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+			$wp_query = new \WP_Query(); // phpcs: ignore WordPress.WP.GlobalVariablesOverride.Prohibited.
```

Comment:
Later, it'd be nice to add a unit test for `block_template()`, as that's a really important method. You already added a really nice test for `test_enqueue_block_styles()` 😄 

I don't think this has to happen in this PR, though.